news/scripts/news_cluster.spec.py

```diff
@@ -0,0 +1,124 @@
+from unittest.mock import patch
+
+from mamba import description, it
+from sure import expect  # pylint: disable=W0611
+
+from news_cluster import ArticleClustering
+
+DATA = [ { 
```

Comment:
Can we move these constants into a fixtures file?

Reply:
I don't see the utility behind it if the data will only be used in this test suite

Reply:
I just find it obscures the test behavior

Reply:
how?

Reply:
The majority of the file's content is the data, while the tests are lost at the bottom. I thought it would be clearer to separate the data from the expected behavior

Reply:
Well it's a .spec.py file, so it's easily identifiable as a test file. I don't think the behavior gets obscured just by the presence of data.